### PR TITLE
Zns updates

### PR DESCRIPTION
**Pull request by aravind-wdc**
Add documentation for:
Getting started with RocksDB  on ZNS SSDs.
Update filesystem section, about f2fs support for ZNS ssds.

### CODE REVIEW

File: docs/linux/fs.md [hunk 1]
```diff
@@ -491,6 +491,64 @@ setup necessary.
 # mount /dev/sdb /mnt
 ```
 
+### Support for ZNS SSDs.
```
Comment (damien-lemoal) [inline docs/linux/fs.md:494]: No "." at the end of the section title please.
Also add a blank line after the section title.

Comment (damien-lemoal) [reply docs/linux/fs.md:494]: Also, s/ZNS/NVMe ZNS

**Comment by damien-lemoal** [comment]
manually applied with modifications. Thanks.

**Comment by aravind-wdc** [comment]
Hi Damien, 
Thanks for pulling in.

Aravind

**Pull request closed without merging**